### DISCUSSION TITLE
How to add GitHub last commit badge for private repo without shields.io server instance

### DISCUSSION THREAD

**Discussion by jamesbraza** [badges/shields#5917] [Q&A] [answered]
I am trying to set up a "GitHub last commit (branch)" badge for a private repo.

I already have badges for [CircleCI](https://circleci.com/docs/2.0/status-badges/) and [Codecov](https://stackoverflow.com/a/61154285/11163122), both require adding a badge token to the badge URL somewhere.

How can one add a token to the `/github/last-commit` badge for a GitHub private repo?

Additionally, where does one get a badge token for GitHub?  I am having a tough time figuring that out as well.

---

I see from https://github.com/badges/shields/issues/593 that one can:

1. Set up your own instance of a shields server
2. Add tokens to `secret.json`

However, I am hoping to not have to do that, and just be able to add the token to the URL somehow.

**Comment by calebcartwright** [comment, marked as answer]
Thanks for reaching out @jamesbraza but I'm going to close this as a duplicate of both the issue you linked in #593 and the existing discussion topic in https://github.com/badges/shields/discussions/5676.

The Shields.io service cannot be used for private GitHub repositories
